akka-http-core/src/test/scala/akka/http/impl/engine/client/TlsEndpointVerificationSpec.scala

```diff
@@ -21,7 +21,9 @@ import javax.net.ssl.SSLException
 
 class TlsEndpointVerificationSpec extends AkkaSpec("""
     akka.loglevel = DEBUG
-    akka.io.tcp.trace-logging = off""") with ScalaFutures {
+    akka.io.tcp.trace-logging = off
+    akka.http.client.parsing.tls-session-info-header = on
+    akka.http.server.parsing.tls-session-info-header = on""") with ScalaFutures {
```

Comment:
Could also be `akka.http.parsing.tls-session-info-header = on`, right? 
Due to how we resolve config (sanity checking that it works here too).